.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
```

Comment:
[OPTIONAL]- According to the project requirements, the `.gitignore` file should contain a `test.md` file. Please, I'd suggest you include  `test.md` file in your repo and ensure it is ignored by git 👍 👍 

Note: I have made this optional because I noticed you opened a new PR and though it is a project requirement, the previous reviewers skipped it. 

Please have it in mind that in your future project, you are not to open a new PR for the same milestone 😄 😄

Reply:
Alright. Thanks